core/src/main/java/com/amplifyframework/api/graphql/GraphQLResponse.java

```diff
@@ -110,24 +114,79 @@ public int hashCode() {
      * See https://graphql.github.io/graphql-spec/June2018/#sec-Response-Format
      */
     public static final class Error {
+        // Description of the error
         private final String message;
 
+        // list of locations describing the syntax element
+        private final List<GraphQLLocation> locations;
+
+        // Details the key path of the response field with error.
+        private final List<GraphQLPathSegment> path;
+
+        // Additional error map, reserved for implementors to use however they see fit.
+        private final Map<String, Object> extensions;
+
         /**
          * Constructs error response in accordance with GraphQL specs.
          * @param message error message
+         * @param locations list of locations describing the syntax element
+         * @param path The key path of the response field with error.
+         * @param extensions additional error map, reserved for implementors.
          */
-        public Error(@NonNull String message) {
+        public Error(@NonNull String message,
+                     @Nullable List<GraphQLLocation> locations,
+                     @Nullable List<GraphQLPathSegment> path,
+                     @Nullable Map<String, Object> extensions) {
             this.message = Objects.requireNonNull(message);
+            this.locations = locations;
+            this.path = path;
+            this.extensions = extensions;
         }
 
         /**
          * Gets the error message.
+         *
          * @return error message
          */
+        @NonNull
         public String getMessage() {
             return message;
         }
 
+        /**
+         * Gets the list of locations where each location describes the beginning of an associated
+         * syntax element.
+         *
+         * @return locations
+         */
+        @Nullable
+        public List<GraphQLLocation> getLocations() {
+            return Immutable.of(locations);
+        }
+
+        /**
+         * Gets the key path of the response field which experienced the error.  This allows clients
+         * to identify whether a null result is intentional or caused by a runtime error.  The
+         * values are either strings or 0-index integers.
+         *
+         * @return path
+         */
+        @Nullable
+        public List<GraphQLPathSegment> getPath() {
+            return Immutable.of(path);
+        }
+
+        /**
+         * Returns additional error information of type E.  Reserved for GraphQL implementors to
```

Comment:
Whoops I think the JavaDoc is left from a prior pre-commit factoring.

Reply:
Yep, fixed!